install-VS2019.bat

```diff
@@ -1,28 +1,51 @@
-@echo off
-
-cd /D "%~dp0"
-
-if not exist "%ProgramFiles(x86)%" (
-  set "ProgramFiles(x86)=%ProgramFiles%"
-)
-
-set "VSWhere=%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe"
-
-:MSBuild
-echo Looking for MSBuild
-for /f "usebackq delims=" %%i in (`^""%VSWhere%" -latest -prerelease -version "[16.0,17.0)" -products * -requires "Microsoft.Component.MSBuild" -property "installationPath"^"`) do (
-  set InstallDir=%%i
-)
-
-set "MSBuild=%InstallDir%\MSBuild\Current\Bin\MSBuild.exe"
-
-if not exist "%MSBuild%" (
-  echo MSBuild not found. Please make sure Visual Studio 16.0+ is installed.
-  pause
-  goto:eof
-)
-
-echo Building Builder.sln
-"%MSBuild%" Builder.sln /nologo /maxcpucount /nodeReuse:false /verbosity:minimal /t:Restore;Build
-
-start "Cosmos Builder" "source\Cosmos.Build.Builder\bin\Debug\Cosmos.Build.Builder.exe" "-VSPATH=%InstallDir%" %*
+@echo off
+setlocal
+for /f "tokens=4-5 delims=. " %%i in ('ver') do set VERSION=%%i.%%j
+
+if "%version%" == "6.1" goto seven

```

Comment:
Should this not check for version != 6.1 and then goto start?

Reply:
it checks if the NT version is 6.1, if yes then it goes to :seven and checks for a specific update, etc

Reply:
My understanding of batch might be wrong, but doesnt the code end up executing the seven block even if the NT version is not 6.1?

Reply:
ah that's true lol, didn't see that one